docs/tutorials/inference/latent_distribution_test.ipynb

```diff
@@ -79,14 +78,16 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Latent distribution test where null is true\n",
+    "## Latent distribution test where null is true\n",
     "We want to know whether the latent positions of the two graphs above were generated from the same latent distribution. In other words, we are testing\n",
     "\n",
     "$$ H_0:F_{X_1} = F_{X_2} R$$$$ H_\\alpha: F_{X_1} \\neq F_{X_2} R $$ \n",
     "\n",
     "The $R$ is an orthogonal rotation matrix present due to the orthogonal non-identifiability in the random dot product graphs.\n",
```

Comment:
i feel that W would be better than R because it's consistent with how we call it throughout the align module and in much of the literature.

Reply:
i had a brain lag: in align we call it Q. that's Agterberg's fault though. i think we should be consistent, but idk what should we call it though. Q? W?

@bdpedigo

Reply:
Q seems fine